### PR TITLE
[zh] Sync manage resources for containers

### PR DESCRIPTION
**Pull request by tengqm**
The manage-compute-resources-container.md file was renamed in the English site. The content in it has been reorganized for *who knows* reasons. This PR fixes the Chinese localization to make things work again.

### CODE REVIEW

File: content/zh/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -0,0 +1,1325 @@
...
+如果 kubelet 没有度量本地临时性存储的用量，即使 Pod
+的本地存储用量超出其约束值也不会被逐出。
+
+不过，如果用于可写入容器镜像层、节点层面日志或者 `emptyDir` 卷的文件系统中可用空间太少，
+节点会为自身设置本地存储不足的{{< glossary_tooltip text="污点" term_id="taint" >}} 标签。
+这一污点会触发对那些无法容忍该污点的 Pods 的逐出操作。
+
+关于临时性本地存储的配置信息，请参考[这里](#configurations-for-local-ephemeral-storage)
+{{< /caution >}}
+
```
Comment (SataQiu) [inline content/zh/docs/concepts/configuration/manage-resources-containers.md:735]: 预览页面，这以后的内容没有显示出来（不确定是否是我浏览器的问题，还请检查下）https://deploy-preview-21903--kubernetes-io-master-staging.netlify.app/zh/docs/concepts/configuration/manage-resources-containers/#

Comment (tengqm) [reply content/zh/docs/concepts/configuration/manage-resources-containers.md:735]: checking.

Comment (tengqm) [reply content/zh/docs/concepts/configuration/manage-resources-containers.md:735]: Got it. My bad. I fixed the preview locally but forgot to 'git push-f'.
Pushing local changes now.

**Comment by tengqm** [comment]
lessons learnt: the `{{tabs}}` shortcode may interfere site build even when placed inside HTML comment, i.e. `<!-- {{ tabs }} -->` is not commenting the shortcode away. Sigh.

**Comment by tengqm** [comment]
> Add some comments

Thanks for the careful review!

**Pull request merged**